src/landing/search_bar.rs

```diff
@@ -94,6 +95,17 @@ live_design! {
                 height: Fit,
                 empty_message: "Search Model by Keyword"
             }
+
+            x_button = <MoxinButton> {
```

Comment:
Another suggestion, we should aim for semantic names that are also consistent with the naming of other elements in the UI.
If we name the button to something like `clear_input_button` or `clear_text_button`, what the element does becomes more obvious.

Reply:
ok，I have improved the code. Thank you very much for your review!